### ISSUE TITLE
[4e8ab6] Element with role attribute has required states and properties - Combobox Failed Example 6

### ISSUE THREAD

**Issue by giacomo-petri** [auto-wcag/auto-wcag#2310]
In rule [Element with role attribute has required states and properties](https://www.w3.org/WAI/standards-guidelines/act/rules/4e8ab6/proposed/), combobox Failed Example 6 requires the aria-controls attribute. 

In [ARIA 1.2 specs](https://www.w3.org/TR/wai-aria-1.2/#combobox) the aria-controls attribute is required, but since aria-controls has no support at all, it has been removed in [ARIA 1.3](https://www.w3.org/TR/wai-aria-1.3/). 

Although this technically violates ARIA 1.2 (until version 1.3 is officially released), requiring support for a property that no UA/AT currently supports seems like overkill for the author to me.

I suggest switching aria-expanded and aria-controls so that the example remains meaningful, aligns with ARIA 1.3 when it's released, and requires no further changes.

**Comment by giacomo-petri** [comment]
I just realised that Failed Example 5 is covering the missing aria-expanded scenario, so the suggestion doesn't work and probably we should simply remove the example.

**Comment by carlosapaduarte** [comment]
CG decision: remove the example

**Comment by WilcoFiers** [comment]
Here's the ARIA ticket that removed the aria-controls requirement from combobox:
https://github.com/w3c/aria/issues/2132

**Comment by giacomo-petri** [comment]
Closed with #2313
